repository/g.json

```diff
@@ -416,7 +416,14 @@
 		{
 			"name": "GhostText",
 			"details": "https://github.com/GhostText/GhostText-for-SublimeText",
+			"issues": "https://github.com/fregante/GhostText/issues",
+			"donate": "https://github.com/sponsors/fregante",
+			"homepage": "https://ghosttext.fregante.com",
+			"author": "fregante",
```

Comment:
Author here means "author of the package", not whatever the underlying tools are. Similarly for issues, it's typical for issues here to mean "issues with the ST package", not "issues with the underlying tool" and it's common practice to point users to other repositories for those underlying tools. Many packages are just thin wrappers around something else. I think your changes here might conflate these things in a way that's not 100% correct and might be confusing. Correct me if I'm wrong though.

Reply:
You might to follow those links to better understand the setup.

- ghosttext/ghosttext-for-sublimetext is the repo for this package, but it does not have issues
- fregante/ghosttext is the main repo, where all the issues (browser extensions and ST package) are opened
- I'm the maintainer of the whole GhostText org, including the ST package, as the original programmer *ghosted* the project several years ago

Reply:
Thanks for explaining! I guess I was mostly reacting to the fact that you don't seem to have many actual commits on the Sublime Text package. Since you're the maintainer of the thing as a whole it's ok to have you as the "author" here, although I would argue that for the ST-specific package it might actually be fairer to have it point to the org, signaling that it's a community project more than something you've created all by yourself.